src/App.jsx

```diff
@@ -11,10 +10,9 @@ import { AddItem, Home, Layout, List } from './views';
 
 import { getItemData, streamListItems } from './api';
 import { useStateWithStorage } from './utils';
-import { useTheme } from './Theme/themeSettings';
+import { ThemeProvider } from './theme/ThemeProvider';
```

Comment:
It probably makes more sense to create a ThemeProvider, rather than a hook. This way, _all_ the theme code is hidden away from App, including the `CSSBaseLine`.